src/3rd_party_add.c

```diff
@@ -180,16 +180,20 @@ enum swupd_code third_party_add_main(int argc, char **argv)
 	if (revert) {
 		/* there was an error adding the repo, revert the action,
 		 * we don't want to keep a corrupt repo */
-		info("There was an error adding 3rd-party repository %s, reverting...\n", name);
 		set_path_prefix(path_prefix);
 		set_state_dir(state_dir);
 		ret = remove_repo(name);
-		if (!ret) {
-			info("Operation reverted\n");
-		} else {
-			error("There was an error removing the repository (errno: %d)\n", ret);
+		if (ret) {
+			error("There was an error removing the corrupt repository (errno: %d)\n\n", ret);
```

Comment:
Getting a little verbose .. how about "error removing the corrupt repository"

Reply:
Hmm, that would look strange cause the error() function is already going to add "Error: ", so to would read " Error: error removing the corrupt repository".
I could change it to something like "The corrupt repository failed to be removed", what do you think?

Reply:
I would put the event first... but upto you.

how abt you?
Failed to remove corrupt repository